_saas-integrations/pardot/pardot 15-10-2015.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Pardot
-permalink: /integrations/saas/pardot
+permalink: /integrations/saas/pardot/v15-10-2015
```

Comment:
I made this change so I could point it out. Don't forget to add the version to the old integration version's permalink, or there'll be two pages with identical URLs.